Sprint-3/revise/implement/is-prime.js

```diff
@@ -0,0 +1,21 @@
+
+function isPrime(num) {
+    // Check if the input is less than 2, which is not prime
+    if (num < 2) return false;
+
+    // Loop from 2 to the square root of the number
```

Comment:
You can possibly improve the performance of the code in the following manners:
- Check if num is 2, and check only odd numbers >= 3 in the loop
- Avoid calling `Math.sqrt(num)` repeatedly by assigning the value of `Math.sqrt(num)` to a variable once, and then refer to the variable in the condition of the loop.
  - Note: The condition is checked at the start of every iteration.